### ISSUE TITLE
problems building under ubuntu 14.04

### ISSUE THREAD

**Issue by unxed** [elfmz/far2l#1502]
```
CMake Error at colorer/CMakeLists.txt:59 (target_link_libraries):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:C>

  Expression did not evaluate to a known generator expression


CMake Error at colorer/src/Colorer-library/src/CMakeLists.txt:215 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:C>

  Expression did not evaluate to a known generator expression


CMake Error at colorer/src/Colorer-library/src/CMakeLists.txt:215 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


```

```
In file included from /root/far2l/far2l/multiarc/src/formats/7z/C/AesOpt.c:65:0:
/usr/lib/gcc/x86_64-linux-gnu/4.8/include/wmmintrin.h:34:3: error: #error "AES/PCLMUL instructions not enabled"
 # error "AES/PCLMUL instructions not enabled"
   ^
```

```
/root/far2l/far2l/multiarc/src/formats/7z/C/AesOpt.c:80:36: error: incompatible types when assigning to type ‘__m128i’ from
type ‘int’
 #define MM_OP(op, dest, src)  dest = op(dest, src);
                                    ^
```

**Comment by unxed** [comment]
Actually not relevant since portable version able to run on Ubuntu 14.04 is available. Build scripts and binary distribution is here:
https://github.com/unxed/far2l-deb/tree/master/portable

Some details on getting portable version working is here: #1505

Full list of build for different distros is here: #647

**Comment by unxed** [comment]
AES problem sould be fixed in #1762

**Comment by unxed** [comment]
Other problems building on Ubuntu 14.04 were:

1. Too old cmake. Solution:
```
sudo apt install -y software-properties-common
sudo add-apt-repository -y ppa:george-edison55/cmake-3.x # cmake 3.0.2+ requred by far2l
sudo apt-get update
sudo apt-get install -y cmake
```

2. Problem with 7z if using gcc8. [Solution](https://github.com/elfmz/far2l/pull/1762#issuecomment-1646820229).

3. No spdlog on Ubuntu 14.04. Already fixed, spdlog is optional now:
https://github.com/elfmz/far2l/pull/1763
https://github.com/elfmz/far2l/pull/1764
https://github.com/elfmz/far2l/pull/1765
https://github.com/elfmz/far2l/pull/1766
